src/app/zap-templates/zcl/zcl-with-test-extensions.json

```diff
@@ -323,6 +324,14 @@
             "OnMode",
             "FeatureMap"
         ],
+        "Microwave Oven Control": [
```

Comment:
This is part of the implementation, not the cluster XML.  Until we have implementation, we should not be adding anything describing that implementation here.